packages/account/src/Types/common.type.ts

```diff
@@ -340,6 +340,7 @@ export type TEmployeeDetailsTinValidationConfig = {
     is_real?: boolean;
     is_tin_auto_set?: boolean;
     is_duplicate_account?: boolean;
+    has_regulated_mt5?: boolean;
```

Comment:
i see we are not checking if user has regulated mt5 account. As discussed can we reuse same store value?
`is_employment_status_tin_mandatory`